addons/survey/models/survey_question.py

```diff
@@ -170,15 +170,17 @@ def default_get(self, fields):
             'All "Is a scored question = True" and "Question Type: Date" questions need an answer')
     ]
 
-    @api.onchange('validation_email')
-    def _onchange_validation_email(self):
-        if self.validation_email:
-            self.validation_required = False
+    @api.depends('validation_email')
+    def _compute_validation_required(self):
+        for question in self:
+            if question.validation_required is None or question.validation_email:
+                self.validation_required = False
 
-    @api.onchange('is_page')
-    def _onchange_is_page(self):
-        if self.is_page:
-            self.question_type = False
+    @api.depends('is_page')
+    def _compute_question_type(self):
+        for question in self:
+            if question.question_type is None or question.is_page:
```

Comment:
Is that "is None" really necessary ? It turns out I had less issues than expected with event. Could be great to try to remove them and see what really happens.